package.json

```diff
@@ -30,7 +31,10 @@
     "mysql2": "^2.2.5"
   },
   "devDependencies": {
+    "@commitlint/cli": "^12.0.1",
+    "@commitlint/config-conventional": "^12.0.1",
     "concurrently": "^5.1.0",
+    "eslint": "^7.23.0",
     "eslint-config-airbnb": "18.2.1",
     "eslint-config-node": "4.1.0",
     "eslint-config-prettier": "^8.1.0",
```

Comment:
@avinashupadhya99 

Can you check from the dev dependencies that I installed earlier that if they are required or not? You could just uninstall some of them if they are not used

Reply:
Sure, I can check that